_posts/2024-02-28-the-return-of-the-golden-man.md

```diff
@@ -0,0 +1,11 @@
+---
+title: The return of the golden <s>child</s> man
+summary: ysbaddaden is back!
+thumbnail: +
+author: beta-ziliani
+release_notes: false
+---
+
+If you been paying attention to what's happening with [multi-threading support in Crystal](/2024/02/09/84codes-manas-mt/), you'll note that a name stands out: Julien Portalier. Also known as [@ysbaddaden](https://github.com/ysbaddaden), he was part of the Crystal Core Team until 2019. Now that he's taken on such a critical task, it only makes sense to bring him back to the Core Team.
```

Comment:
**issue:** Actually, Julien announced his departure to fellow Core Team members on 30 April 2020 (https://forum.crystal-lang.org/t/so-long-and-thanks-for-all-the-fish/2026 - internal link, only visible to Core Team members). If we want to stick a date to it, we should take this one.
Btw. he became a member in September 2015 so we can put a full time span to his previous tenure (https://crystal-lang.org/2015/09/04/crystal-0.7.7-released/).

```suggestion
If you been paying attention to what's happening with [multi-threading support in Crystal](/2024/02/09/84codes-manas-mt/), you'll note that a name stands out: Julien Portalier. Also known as [@ysbaddaden](https://github.com/ysbaddaden), he had been a part of the Crystal Core Team from 2015 to 2020. Now that he's taken on such a critical task, it only makes sense to bring him back to the Core Team.
```

The change to past perfect is just polish. I think it fits better with the perfect tense in the following sentance.